osmaxx/excerptexport/models/export.py

```diff
@@ -49,6 +49,7 @@ def status_update_url(self):
         return reverse('job_progress:tracker', kwargs=dict(export_id=self.id))
 
     def set_and_handle_new_status(self, new_status, *, incoming_request):
+        assert new_status in dict(STATUS_CHOICES)
```

Comment:
`STATUS_CHOICES` could be a nested tuple, in theory, but it practices, this is OK, since we know it isn't, and if it ever should be, we will see this assertion fail :+1: